README.md

```diff
@@ -1 +1,54 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+ this project, is setted up for Learning Purpose .No complex coding is required for this exercise. the goal here is to master all of this tools (git,github workflow and linter).
+
+ ### Built With
+
+ - Html,css
+ - No framework used
+ - Lint
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Please edit this section to better suit your project. if there's a section that you're not using, it's better off to be removed.